src/electron/README.md

```diff
@@ -52,3 +52,12 @@ apt update && apt install -y gcc-mingw-w64-x86-64
 To build the _release_ version of Windows installer, you'll also need:
 
 - [Java 8+ Runtime](https://www.java.com/en/download/). This is required for the cross-platform Windows executable signing tool [Jsign](https://ebourg.github.io/jsign/). If you don't need to sign the executables, feel free to skip this.
+
+## Linux
+
+To build for Linux on a macOS, you need to first install the [musl-cross compiler](https://github.com/GregorR/musl-cross).
```

Comment:
The `brew` package seems to reference https://github.com/richfelker/musl-cross-make instead. We might need to carefully decide which one to use.

We can build out own, or download one of the pre-built binary here: https://musl.cc/ . (Though musl declares it to be unofficial, but it's hosted under the `musl` domain name)

Reply:
I changed to zig, which provides its own official package.